.readthedocs.yml

```diff
@@ -1,15 +1,24 @@
 ---
 # Read the Docs configuration file
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+#
+# The pre_build command checks if we're building a named branch (i.e., not a PR).
+# If not, check out doc/releases from the main branch so it's always up to date
```

Comment:
At least the way I read it, this comment doesn't agree with the code (and what you said earlier in the PR comments).  To double check:
- named branch --> not a PR build --> substitute `doc/releases` from `main`
- detached HEAD --> PR build --> leave as is

If the above is the intended behavior, the code is correct but this comment could use an adjustment:
```suggestion
# If so, check out doc/releases from the main branch so it's always up to date
```

Reply:
![](https://media.giphy.com/media/jUwpNzg9IcyrK/giphy.gif)